package.json

```diff
@@ -37,7 +37,7 @@
     "eth-ens-namehash": "^2.0.8",
     "eth-gas-reporter": "^0.1.1",
     "ethereumjs-abi": "^0.6.5",
-    "ganache-cli": "^6.4.2",
+    "ganache-cli": "~6.2.0",
```

Comment:
Interesting, was the latest version failing due to the gas estimation?

Reply:
Exactly, I was facing this [issue](https://github.com/trufflesuite/ganache-core/issues/417)